deploy/backplane/acs/01-acs-admins-project.ClusterRole.yaml

```diff
@@ -45,6 +45,26 @@ rules:
   - get
   - list
   - watch
+# ACS Team can view ACS Addon CRs
+- apiGroups:
+  - package-operator.run
+  - addons.managed.openshift.io
```

Comment:
i think the rules should be separated by apigroup so it's clear which resources belong to the `package-operator.run` group and which ones belong to `addons.managed.openshift.io`

Reply:
Splitted it into two groups